### PR TITLE
update: Avatar feature created

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
I have created an Avatar feature it has a Default avatar, circular avatar, and avatar with a border

### CODE REVIEW

**Review by tushartiwari7 — APPROVED** [review]
Code is easy to understand.
Yes you can merge this

File: index.html [hunk 1]
```diff
@@ -28,9 +28,48 @@
...
                 <header class="section-header">Avatar</header>
-                <article class="container-info"></article>
+                <article class="container-info">
+                    <div class="info">
+                        <h4> Avatar is used to represent a user or Profile picture</h4></div>
+                    <div class="avatar-container">
+                        <div class="text-badge">Default Avatar</div>
+                        <div class="avatar">
+                            <div class="image">
+                                <img src="image.jpg" alt="sunset" class="avatar-image">
```
Comment (barbaricankit) [inline index.html]: img is a self-closing tag, how is this working for you. Please check

**Review by sks-21 — APPROVED** [review]
Looks good to merge.

File: index.html [hunk 2]
```diff
@@ -28,9 +28,48 @@
...
+                        </div>
+                        <div class="code">
+                            <iframe src="https://www.thiscodeworks.com/embed/61f8e6601e1df40015379388" style="width: 100%; height: 151px;" frameborder="0"></iframe>
+                        </div>
+                    </div>
+                    <div class="avatar-container">
+                        <div class="text-badge">Circular Avatar</div>
+                        <div class="avatar">
+                            <div class="image">
+                                <img src="image.jpg" alt="sunset" class="avatar-image img-round">
```
Comment (barbaricankit) [inline index.html]: at one place you are giving the class name as avatar-image and in another one giving img-round. It should be consistent, better to have both as img

File: style.css [hunk 3]
```diff
@@ -381,6 +381,39 @@ body {
...
+}
+.avatar-image{
+    height:50px;
+    width:50px;
+}
+.avatar{
+    padding:1rem;
+    display:flex;
+    flex-direction:column;
+    /* justify-content: center; */
```
Comment (barbaricankit) [inline style.css]: remove unnecessary comments from code.

File: style.css [hunk 4]
```diff
@@ -381,6 +381,39 @@ body {
...
+    align-items: center;
+}
+.img-round{
+    border-radius: 50% !important;
```
Comment (barbaricankit) [inline style.css]: why use important in this place? You have very less code so how come you have overriding properties that you need to use important

File: style.css [hunk 5]
```diff
@@ -381,6 +381,39 @@ body {
 .shadow-cont{
     box-shadow: 5px 5px 5px 5px var(--off-white);
 }
+
+/* ------------avatar-feature-------------- */
+.avatar-container{
+    padding:1rem;
+    display:flex;
+    flex-direction:column;
```
Comment (barbaricankit) [inline style.css:350]: You can give a class name as flex-col and have the code in that class and just use that class everywhere.

**Review by barbaricankit — COMMENTED** [review]
Looks good. I have provided some suggestions, let me know if you have any issues

File: index.html (hunk 2 above)
Comment (Kothalkarpriya) [reply index.html]: I have changes the class name to round-image and also similar changes made

File: style.css (hunk 5 above)
Comment (Kothalkarpriya) [reply style.css:350]: I will use it. Thank you

**Pull request merged**